velox/exec/HashTable.cpp

```diff
@@ -57,6 +57,7 @@ HashTable<ignoreNullKeys>::HashTable(
       pool_(pool),
       minTableSizeForParallelJoinBuild_(minTableSizeForParallelJoinBuild),
       isJoinBuild_(isJoinBuild),
+      joinBuildNoDuplicates_(!allowDuplicates),
```

Comment:
joinBuildNoDuplicates_ -> joinBuildDistinct

Reply:
Do you think I need to change all instances of "NoDup" to "Distinct"? There are several variables that use "NoDup," including abandonBuildNoDupHashMinRows_, abandonBuildNoDupHashMinPct_, and others. The reason I use "NoDup" is that HashTable has the concept of "drop duplicates".

Reply:
Sounds reasonable